petrophys/visualization/visualize.py

```diff
@@ -0,0 +1,56 @@
+import matplotlib.pyplot as plt
+import matplotlib as mpl 
+
+def valtonan(inp, val=-999.25):
+    """Convert all 'val' to NaN's."""
+    inp[inp==val] = np.nan
+    return inp
+
+#This function makes for cleaner axis plotting
+def remove_last(ax, which='upper'):
+    """Remove <which> from x-axis of <ax>.
```

Comment:
Would be good to add the input and output into your docstring
```suggestion
    """Remove <which> from x-axis of <ax>.
    
    Parameters
    ------------
    which : str
    <description of variable> 
```

Reply:
I have a new commit please have a look

Reply:
Looks good